setup.py

```diff
@@ -36,7 +65,6 @@ def get_version(*file_paths):
 
 
 
-requirements = ['Click>=7.0', ]
```

Comment:
this is being installed inside `base.in` but please whoever reviews this PR. Please confirm if the the removal of  line 39 and line 64 `install_requires=requirements` is correct